test/screenshot/index.hbs

```diff
@@ -1,15 +1,16 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
does this have to be a handlebars file? Can we just serve a straight HTML file and be done?